diff --git a/dinov2/layers/attention.py b/dinov2/layers/attention.py
index f1d3dab..72218ba 100644
--- a/dinov2/layers/attention.py
+++ b/dinov2/layers/attention.py
@@ -7,6 +7,8 @@
 #   https://github.com/facebookresearch/dino/blob/master/vision_transformer.py
 #   https://github.com/rwightman/pytorch-image-models/tree/master/timm/models/vision_transformer.py
 
+from __future__ import annotations
+
 import logging
 import os
 import warnings
diff --git a/dinov2/layers/block.py b/dinov2/layers/block.py
index 7e83b71..7a7776c 100644
--- a/dinov2/layers/block.py
+++ b/dinov2/layers/block.py
@@ -7,6 +7,8 @@
 #   https://github.com/facebookresearch/dino/blob/master/vision_transformer.py
 #   https://github.com/rwightman/pytorch-image-models/tree/master/timm/layers/patch_embed.py
 
+from __future__ import annotations
+
 import logging
 import os
 from typing import Callable, List, Any, Tuple, Dict, Optional
